### PR TITLE
Changing from old STM CLA to the new DCO

### PR DESCRIPTION
**Pull request by jbech-linaro**
`Signed-off-by: Joakim Bech <joakim.bech@linaro.org>`

### CODE REVIEW

**Comment by pascal-brand-st-dev** [comment]
Once op-tee[at]linaro[.]org exists:

    Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

File: Notice.md [hunk 1]
```diff
@@ -1,35 +1,103 @@
 OP-TEE
 =======
 
-This is the repository of OP-TEE (Open Portable Trusted Execution Environment), the open-source TEE maintained by STMicroelectronics, with initial contributions from STMicroelectronics, Ericsson, the Linaro industry association.
+This is the repository of OP-TEE (Open Portable Trusted Execution Environment),
+the open-source TEE maintained by Linaro, with initial contributions from
+STMicroelectronics, Ericsson and the Linaro Limited.
```
Comment (jenswi-linaro) [inline Notice.md]: "the Linaro Limited"?

Comment (jbech-linaro) [inline Notice.md]: Will fix.

**Comment by jbech-linaro** [comment]
@pascal-brand-st-dev I we can merge this without having op-tee@ ... ready, since I'd imagine that it's not very often we have gotten email on that particular mail address? Having that said, I will ask IS/IT to setup that mail in the beginning of next week.

**Comment by pascal-brand-st-dev** [comment]
Reviewed-by: Pascal Brand <pascal.brand@linaro.org>

**Comment by jenswi-linaro** [comment]
```
Reviewed-by: Jens Wiklander <jens.wiklander@linaro.org>
```

**Comment by jbech-linaro** [comment]
Rebased and tags applied.

**Pull request merged**